src/users/repositories/user.repository.ts

```diff
@@ -36,20 +36,20 @@ export class UserRepository {
     ).rank;
   }
 
-  async findUser(email: string, provider: string): Promise<User | undefined> {
+  findUser(email: string, provider: string): Promise<User | undefined> {
     return this.entityManager.findOne(User, { where: { email, provider } });
   }
 
-  async createUser(entityManager: EntityManager, userInfo: any): Promise<User> {
-    const user = new User();
-    user.provider = userInfo.provider;
-    user.name = userInfo.nickname;
-    user.email = userInfo.email;
-    user.hopeCategoryId = 1;
-    user.activityCategoryId = 1;
-    user.isMentor = false;
-
-    return entityManager.save(user);
+  createUser(entityManager: EntityManager, userInfo: any): Promise<User> {
+    const { provider, nickname: name, email } = userInfo;
+    return entityManager.save(User, {
+      provider,
+      name,
+      email,
+      hopeCategoryId: 1,
+      activityCategoryId: 1,
+      isMentor: false,
+    });
   }
```

Comment:
```suggestion
  createUser(entityManager: EntityManager, userInfo: any): Promise<User> {
    const { provider, nickname: name, email } = userInfo;
    return entityManager.save(User, {
      ...userInfo
      hopeCategoryId: 1,
      activityCategoryId: 1,
      isMentor: false,
    });
  }
```
프로퍼티 이름을 name으로 하면 이렇게 깔끔해지겠네요
그리고 userInfo의 타입을 따로 만들어 놓아서 관리하는게 더 좋을 것 같아요